charts/tempo-distributed/values.yaml

```diff
@@ -224,6 +224,55 @@ ingester:
   appProtocol:
     # -- Set the optional grpc service protocol. Ex: "grpc", "http2" or "https"
     grpc: null
+  zoneAwareReplication:
+    # -- Enable zone-aware replication for ingester
+    enabled: true
```

Comment:
I don't know what to expect if we go from default `false` to default `true` here.  Perhaps a default `false` makes sense here.

Reply:
yeah, make sense

Reply:
In my testing, I had to fix the ring after destroying the old instances, one remained unhealthy in the ring and had to forget it.  Once I dropped that member, everything looks pretty good.

Reply:
its possibe we need to use the `-ingester.ring.unregister-on-shutdown`